docs/en/enterprise-edition/policy-reference/azure-policies/azure-logging-policies/bc-azure-2-53.adoc

```diff
@@ -0,0 +1,50 @@
+
+== Azure Synapse Workspace does not have extended audit logs
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/graph_checks/azure/SynapseWorkspaceHasExtendedAuditLogs.yaml[CKV2_AZURE_53]
+
+|Severity
+|INFO
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan
+
+|===
+
+=== Description
+
+This policy ensures that an Azure Synapse Workspace has extended audit logs enabled. Extended audit logs are important because they provide detailed records of access and activity within the workspace. Without these logs, it can be challenging to monitor and investigate any unauthorized access or anomalies, thus posing a security risk. Logging mechanisms are essential for tracking user actions, policy breaches, data access, and troubleshooting issues, making it a critical practice for maintaining security and compliance within Azure environments.
```

Comment:
- This policy detects whether extended audit logs are disabled on an Azure Synapse Workspace. Logging mechanisms are essential for tracking user actions, policy breaches, data access, and troubleshooting issues, making it a critical practice for maintaining security and compliance within Azure environments.
Extended audit logs provide detailed records of access and activity within the workspace, which are crucial for monitoring and investigating potential security risks like unauthorized access or anomalies.
- This policy detects whether extended audit logs are disabled on an Azure Synapse Workspace. Extended audit logs provide detailed records of access and activity within the workspace, which are crucial for monitoring and investigating potential security risks such as unauthorized access or anomalies. 
Logging mechanisms are essential for tracking user actions, policy breaches, data access, and troubleshooting issues, making it a critical practice for maintaining security and compliance within Azure environments.